GatinhoInfo/Controllers/EntranceViewController.swift

```diff
@@ -11,11 +11,44 @@ import Alamofire
 
 class EntranceViewController: UIViewController {
    
-    @IBOutlet weak var checkButton: UIButton!
+    let labelName = UILabel()
+    let checkCatsButton = UIButton()
+    let catDrawing = UIImageView(image:#imageLiteral(resourceName: "catDrawingImage"))
+    let stackView = UIStackView()
+
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        checkButton.layer.cornerRadius = 20
+        
+        view.backgroundColor = #colorLiteral(red: 0.662745098, green: 0.8705882353, blue: 0.9764705882, alpha: 1)
+        
+        labelName.text = "CatInfo"
+        labelName.textAlignment = .center
+        labelName.textColor =  #colorLiteral(red: 0, green: 0, blue: 0, alpha: 1)
+        labelName.font = UIFont(name: "AppleSDGothicNeo-Bold" , size: 50)
+        labelName.layer.cornerRadius = 10
+        labelName.layer.masksToBounds = true
+        
+        checkCatsButton.setTitle("Check Cats", for: .normal)
+        checkCatsButton.backgroundColor = #colorLiteral(red: 0, green: 0, blue: 0, alpha: 1)
+        checkCatsButton.titleLabel?.textColor = #colorLiteral(red: 1.0, green: 1.0, blue: 1.0, alpha: 1.0)
+        checkCatsButton.titleLabel!.font = UIFont(name: "AppleSDGothicNeo-Bold" , size: 25)
+        checkCatsButton.layer.cornerRadius = 5
+        checkCatsButton.addTarget(self, action: #selector(self.buttonClicked(sender:)), for: .touchUpInside)
```

Comment:
Se o `self.` não for obrigatório, pode ser removido.
Verificar no restante da classe.